_config/pom.yml

```diff
@@ -51,28 +51,28 @@ latestFinal:
 
 # latest.version can be equal to latestFinal.version
 latest:
-    version: 7.0.0.Beta7
-    releaseDate: 2017-03-07
+    version: 7.0.0.Beta8
+    releaseDate: 2017-03-30
 
-    droolsZip: https://download.jboss.org/drools/release/7.0.0.Beta7/drools-distribution-7.0.0.Beta7.zip
+    droolsZip: https://download.jboss.org/drools/release/7.0.0.Beta8/drools-distribution-7.0.0.Beta8.zip
 
-    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.0.0.Beta7/droolsjbpm-integration-distribution-7.0.0.Beta7.zip
+    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.0.0.Beta8/droolsjbpm-integration-distribution-7.0.0.Beta8.zip
 
-    droolsWorkbenchEAP7WAR: https://download.jboss.org/drools/release/7.0.0.Beta7/kie-drools-wb-7.0.0.Beta7-eap7.war
-    droolsWorkbenchWildFlyWAR: https://download.jboss.org/drools/release/7.0.0.Beta7/kie-drools-wb-7.0.0.Beta7-wildfly10.war
-    droolsWorkbenchTomcatWAR: https://download.jboss.org/drools/release/7.0.0.Beta7/kie-drools-wb-7.0.0.Beta7-tomcat7.war
-    droolsWorkbenchJcr2vfsZip: https://download.jboss.org/drools/release/7.0.0.Beta7/drools-wb-jcr2vfs-distribution-7.0.0.Beta7.zip
-    droolsWorkbenchExampleGitReposZip: https://download.jboss.org/drools/release/7.0.0.Beta7/kie-wb-example-repositories-7.0.0.Beta7.zip
+    droolsWorkbenchEAP7WAR: https://download.jboss.org/drools/release/7.0.0.Beta8/kie-drools-wb-7.0.0.Beta8-eap7.war
+    droolsWorkbenchWildFlyWAR: https://download.jboss.org/drools/release/7.0.0.Beta8/kie-drools-wb-7.0.0.Beta8-wildfly10.war
+    droolsWorkbenchTomcatWAR: https://download.jboss.org/drools/release/7.0.0.Beta8/kie-drools-wb-7.0.0.Beta8-tomcat8.war
+    droolsWorkbenchJcr2vfsZip: https://download.jboss.org/drools/release/7.0.0.Beta8/drools-wb-jcr2vfs-distribution-7.0.0.Beta8.zip
+    droolsWorkbenchExampleGitReposZip: https://download.jboss.org/drools/release/7.0.0.Beta8/kie-wb-example-repositories-7.0.0.Beta8.zip
 
-    droolsjbpmToolsZip: https://download.jboss.org/drools/release/7.0.0.Beta7/droolsjbpm-tools-distribution-7.0.0.Beta7.zip
+    droolsjbpmToolsZip: https://download.jboss.org/drools/release/7.0.0.Beta8/droolsjbpm-tools-distribution-7.0.0.Beta8.zip
 
-    documentationHtmlSingle: https://docs.jboss.org/drools/release/7.0.0.Beta7/drools-docs/html_single/index.html
+    documentationHtmlSingle: https://docs.jboss.org/drools/release/7.0.0.Beta8/drools-docs/html_single/index.html
 
-    KIE_API_documentationJavadoc: https://docs.jboss.org/drools/release/7.0.0.Beta7/kie-api-javadoc/index.html
+    KIE_API_documentationJavadoc: https://docs.jboss.org/drools/release/7.0.0.Beta8/kie-api-javadoc/index.html
 
-    droolsReleaseNotes: https://docs.jboss.org/drools/release/7.0.0.Beta7/drools-docs/html/ch02.html
+    droolsReleaseNotes: https://docs.jboss.org/drools/release/7.0.0.Beta8/drools-docs/html/ch02.html
```

Comment:
This is a dead link (it was dead even for Beta7, but I think we should fix this anyway).